### PR TITLE
Update CVA docs and fixes

### PR DESCRIPTION
**Pull request by WebMamba**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Issues        | 
| License       | MIT

@weaverryan asked for a few changes here: https://github.com/symfony/ux/pull/1416.
So here are the fixes. 😁

### CODE REVIEW

File: src/TwigComponent/doc/index.rst [hunk 1]
```diff
@@ -1131,29 +1137,17 @@ https://github.com/tales-from-a-dev/twig-tailwind-extra
...
-        }
+       // ...
     }) %}
 
     <div class="{{ alert.apply({color, size}, attributes.render('class')) | tailwind_merge }}">
          {% block content %}{% endblock %}
     </div>
 
-Compounds variants
+Compounds Variants
```
Comment (kbond) [inline src/TwigComponent/doc/index.rst]:
```suggestion
Compound Variants
```

File: src/TwigComponent/doc/index.rst [hunk 2]
```diff
@@ -1131,29 +1137,17 @@ https://github.com/tales-from-a-dev/twig-tailwind-extra
...
+       // ...
     }) %}
 
     <div class="{{ alert.apply({color, size}, attributes.render('class')) | tailwind_merge }}">
          {% block content %}{% endblock %}
     </div>
 
-Compounds variants
+Compound Variants
 ~~~~~~~~~~~~~~~~~~
```
Comment (kbond) [inline src/TwigComponent/doc/index.rst]:
```suggestion
~~~~~~~~~~~~~~~~~
```

**Review by kbond — APPROVED** [review]

File: src/TwigComponent/doc/index.rst [hunk 3]
```diff
@@ -1061,11 +1061,15 @@ Exclude specific attributes:
 Component with Complex Variants (CVA)
 -------------------------------------
 
-CVA (Class Variant Authority) is a concept from the JS world (https://cva.style/docs/getting-started/variants).
-It's a concept used by the famous shadcn/ui library (https://ui.shadcn.com).
+.. versionadded:: 2.16
+
+    The ``CVA`` function was added in TwigComponents 2.16.
+
+[CVA (Class Variant Authority)](https://cva.style/docs/getting-started/variants) is a concept from the JavaScript
```
Comment (javiereguiluz) [inline src/TwigComponent/doc/index.rst]: In `rst` the links use a syntax different than Markdown. This should be:

```rst
`CVA (Class Variant Authority)`_ is a concept from the JavaScript world and used
by the well-known `shadcn/ui`_.
```

And add this at the bottom of the page:

```
.. _`CVA (Class Variant Authority)`: https://cva.style/docs/getting-started/variants
.. _`shadcn/ui`: https://ui.shadcn.com
```

File: src/TwigComponent/doc/index.rst [hunk 4]
```diff
@@ -1108,18 +1114,18 @@ In the variants key you define the different variants of your component.
...
 CVA and Tailwind CSS
 ~~~~~~~~~~~~~~~~~~~~
 
-CVA work perfectly with tailwindcss. The only drawback is you can have class conflicts,
-to have a better control you can use this following bundle (
-https://github.com/tales-from-a-dev/twig-tailwind-extra
-) in addition to the cva function:
+CVA work perfectly with Tailwind CSS. The only drawback is that you can have class conflicts.
+To "merge" conflicting classes together and keep only the one you need, use the
+``tailwind_merge()` method from [tales-from-a-dev/twig-tailwind-extra](https://github.com/tales-from-a-dev/twig-tailwind-extra)
```
Comment (javiereguiluz) [inline src/TwigComponent/doc/index.rst]:
```md
[tales-from-a-dev/twig-tailwind-extra](https://github.com/tales-from-a-dev/twig-tailwind-extra)
```

should be:

```rst
`tales-from-a-dev/twig-tailwind-extra`_

and add this to the bottom of the page:
.. _`tales-from-a-dev/twig-tailwind-extra`: https://github.com/tales-from-a-dev/twig-tailwind-extra
```

File: src/TwigComponent/doc/index.rst (hunk 3 above)
Comment (weaverryan) [reply src/TwigComponent/doc/index.rst]: That was my fault! Thanks for reminding me :)

**Review by weaverryan — APPROVED** [review]
I made a few tweaks - thanks for the fast PR!

**Comment by weaverryan** [comment]
Thanks Matheo!

**Pull request merged**